lib/iris/experimental/ugrid.py

```diff
@@ -71,6 +77,8 @@
 #: Numpy "edgeitems" printoptions default argument.
 NP_PRINTOPTIONS_EDGEITEMS = 2
 
+# VTK slider widget callback hook that requires global context
+VTK_SLIDER_CALLBACK = dict()
```

Comment:
One general point  : could you move all this plotting stuff into a sub-package ?
I think experimental.ugrid is already looking crowded + hard to navigate, 
and this new code seem pretty much self-contained.

Reply:
Yeah, I could do.